tests/Benchmarks/Buffers/Writer.cs

```diff
@@ -11,9 +11,8 @@
 using System.Runtime.CompilerServices;
 using System.Text;
 using System.Text.Utf8;
-using System.Threading;
 
-public class BufferWriterBench
+public class Writer
```

Comment:
Why this change? Isn't "Writer" a bit too generic of a name?

Reply:
I believe the objective is to reduce duplication within the naming. The folder structure already contains "Buffers", "Benchmarks". @adamsitnik, please clarify if I am mistaken.

Reply:
@ahsonkhan you are right. The project is called `Benchmarks`, the subfolder is called `Buffers` so I don't want to duplicate the names in full name

`Benchmarks.Buffers.Writer` vs
`Benchmarks.BufferWriterBench`

I think it's cleaner solution but it's just my personal preference so I can change it if you want @KrzysztofCwalina

Reply:
I am fine with the naming you have. Just wanted to make sure the change is what we as a team like.